labs/yocto-first-build-beagleplay/yocto-first-build-beagleplay.tex

```diff
@@ -27,24 +30,26 @@ \section{Download Yocto}
 \begin{bashinput}
 git clone https://git.yoctoproject.org/git/poky
 cd $HOME/__SESSION_NAME__-labs/poky
-git checkout -b kirkstone-4.0.17 kirkstone-4.0.17
+git checkout -b scarthgap-5.0.4 scarthgap-5.0.4
 \end{bashinput}
 
 Return to your project root directory (\code{cd $HOME/__SESSION_NAME__-labs/})
 and download the \code{meta-openembedded}, \code{meta-arm} and \code{meta-ti} layers:
 \begin{bashinput}
 cd $HOME/__SESSION_NAME__-labs
-git clone -b kirkstone https://git.openembedded.org/meta-openembedded
+git clone -b scarthgap https://git.openembedded.org/meta-openembedded
 git clone https://git.yoctoproject.org/git/meta-arm
 cd meta-arm
-git checkout -b yocto-4.0.3 yocto-4.0.3
+git checkout -b yocto-5.0.1 yocto-5.0.1
```

Comment:
Wouldn't it be better to checkout the same version for both poky and meta-arm? I.e. 5.0.1 for both I guess?

Reply:
meta-arm has not updated their layer to 5.0.4 yet, so I chose to still keep latest poky so that when they do update there should be a low risk of breaking things when updating meta-arm . Now, this might not give the best example, I don't know. What do you think?